models/CoronSegmentator/configs/metadata.json

```
@@ -0,0 +1,88 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "name": "coronaryArtery_ct_segmentation",
+    "version": "1.0.0",
+    "monai_version": "1.3.2",
+    "pytorch_version": "1.4.0",
+    "numpy_version": "2.3.1",
+    "required_packages_version": {
+        "fire": "0.7.0",
+        "monai": " 1.4.0",
+        "nnunetv2": "2.6.0",
+        "nibabel": "5.3.2",
+        "numpy": "2.3.1",
```

Could you please double check the monai version you are using, the one mentioned above is "1.4.0"? Thanks!
```suggestion
        "nnunetv2": "2.6.0",
        "nibabel": "5.3.2",
```